stdlib/stdlib.mli

```diff
@@ -51,10 +51,40 @@ exception Exit
 (** The [Exit] exception is not raised by any library function.  It is
     provided for use in your programs. *)
 
-val try_finally : (unit -> 'a) -> (unit -> unit) -> 'a
-(** [try_finally f g] calls [f ()] and returns its result.  If it raises,
-    the same exception is raised unless [g ()] itself raises;
-    in {b: any} case, [g ()] is called after [f ()] terminates.
+val try_finally :
+  ?always:(unit -> unit) ->
+  ?exceptionally:(unit -> unit) ->
+  (unit -> 'a) -> 'a
+(** [try_finally work ~always ~exceptionally] is designed to run code
```

Comment:
The argument order should be the same in the signature and in the documentation.

Reply:
Oh yes, wel spotted!